src/styles/index.scss

```diff
@@ -0,0 +1,40 @@
+@import url("https://fonts.googleapis.com/css?family=Libre+Franklin:400,700");
+@import url("https://s3.amazonaws.com/tds-static/fonts/moregothic/1.0.0/More+Gothic+Bold.css");
+
+$red:                         #FF4630;
+$purple:                      #DAC4F5;
+$green:                       #D6FFB8;
+$yellow:                      #FFED9C;
+
+$body-bg:                     #FAF9F5;
+
+$font-family-sans-serif:      "Libre Franklin",sans-serif;
+$font-family-monospace:       "More Gothic Bold", Impact, sans-serif;
+
+// Grid needs to be set in px
+$grid-gutter-width:           40px;
+
+// this will allow us to create better responsive font sizes
+$display1-size:               6em;
```

Comment:
Are we using these?

Reply:
Are you talking about the `em` units? or the variable?
The variable I am using only the `display2` but by default bootstrap uses another type of units and I would like to preserve the display behaviour over the entire site so further developers doesn't have to stumble with a surprise of some display text behaving different